integrations/jenkins/README.md

```diff
@@ -12,6 +12,8 @@ Please follow the regular [installation instructions](https://www.jenkins.io/doc
 
 Also ensure that [Docker is installed](https://docs.docker.com/engine/install/) on your system with [BuildKit enabled](https://docs.docker.com/develop/develop-images/build_enhancements/#to-enable-buildkit-builds), and that Jenkins is allowed to run Docker (you might need to run `usermod -aG docker jenkins` to add the `jenkins` user to the `docker` group).
 
+If you're running Jenkins as a docker image as opposed to using the Java WAR, you need to [install docker inside the Jenkins container](https://docs.docker.com/engine/install/debian/).
```

Comment:
Nits:
- In docs, we prefer to spell out "you're" as "you are".
- "docker" -> "Docker" (two times).
- You're referring to Debian-specific install instructions, but we should stay OS-agnostic as much as possible. So probably just use https://docs.docker.com/engine/install/ (without the trailing "debian/") instead.

Reply:
1 and 2 are fine but 3 relates to the fact that the official Jenkins Docker images use Debian specifically. 
Jenkins has always provided docker images based on Debian from the oldest available images on Docker hub.
I'd argue this is more user friendly?

Reply:
> Jenkins has always provided docker images based on Debian

Ok, makes sense then!